### DISCUSSION TITLE
How do you deal with coffee that goes straight into the trashbin?

### DISCUSSION THREAD

**Discussion by nils-werner** [graphefruit/Beanconqueror#256] [Q&A] [answered]
My grinder has about 5g of retention, so if I haven't had some coffee in a few days I will usually grind about 5g of beans and throw the result straight into the trashbin. Only then will I start measuring my beans, grinding them, brewing a coffee etc.

Problem is: Beanconqueror now thinks that I have 5g too many beans left over.

How do you deal with that? Do you create a "trashbin" brewing method? Do you just ignore it?

**Comment by graphefruit** [comment, marked as answer]
Hello @nils-werner,
thanks for the question.
In the next version update 6.0, there will be a new field: Bean weight in.
![image](https://user-images.githubusercontent.com/6260244/132121285-42fcfdf4-3498-4345-a275-634d3600ce72.png)

Therefore you can say: I put in 18 grams beans, and just ground 13 grams in the end, so 5 grams would be waste. 

To discuss your use case: Normally you do 18 grams for your espresso , and you didn't make  coffee some days now:
You put in 23grams as beans weight in and put in 18 grams grounded beans.

The 23grams are taken for the calculation and the statistics.
If you leave this field open / empty, the 18 grams will be taken.

Greetings
Lars

**Reply by nils-werner** [reply]
This is brilliant, thanks!

**Comment by Starglen** [comment]
Thanks for the question and the solution! This is actually a really common pain point for anyone with high-retention grinders. The upcoming "Bean weight in" field in v6.0 sounds like the perfect fix—simple and intuitive. Being able to input what actually goes into the grinder while recording what comes out solves the waste tracking problem without having to create a fake "trashbin" brewing method. Looking forward to it!
